pkgs/tools/misc/staruml/default.nix

```
@@ -30,25 +39,23 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir $out
-    mv opt/staruml $out/bin
+    mv opt/StarUML $out/bin
 
     mkdir -p $out/lib
     ln -s ${stdenv.cc.cc.lib}/lib/libstdc++.so.6 $out/lib/
     ln -s ${lib.getLib systemd}/lib/libudev.so.1 $out/lib/libudev.so.0
 
-    for binary in StarUML Brackets-node; do
-      ${patchelf}/bin/patchelf \
-        --interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-        $out/bin/$binary
-      wrapProgram $out/bin/$binary \
-        --prefix LD_LIBRARY_PATH : $out/lib:${LD_LIBRARY_PATH}
-    done
+    ${patchelf}/bin/patchelf \
```

```suggestion
    patchelf \
```
and add patchelf to nativeBuildInputs.